pyuvdata/beamfits.py

```diff
@@ -506,36 +516,41 @@ def write_beamfits(self, filename, run_check=True,
 
         # check for frequency-specific optional arrays. If they're not None,
         # make a binary table HDU to hold them
-        if (self.system_temperature_array is not None or self.loss_array is not None or
-                self.mismatch_array is not None or self.s_parameters is not None):
-            col_list = []
-            if self.system_temperature_array is not None:
-                sys_temp_col = fits.Column(name='sys_temp', format='D',
-                                           array=self.system_temperature_array[0, :])
-                col_list.append(sys_temp_col)
-            if self.loss_array is not None:
-                loss_col = fits.Column(name='loss', format='D',
-                                       array=self.loss_array[0, :])
-                col_list.append(loss_col)
-            if self.mismatch_array is not None:
-                mismatch_col = fits.Column(name='mismatch', format='D',
-                                           array=self.mismatch_array[0, :])
-                col_list.append(mismatch_col)
-            if self.s_parameters is not None:
-                s11_col = fits.Column(name='s11', format='D',
-                                      array=self.s_parameters[0, 0, :])
-                s12_col = fits.Column(name='s12', format='D',
-                                      array=self.s_parameters[1, 0, :])
-                s21_col = fits.Column(name='s21', format='D',
-                                      array=self.s_parameters[2, 0, :])
-                s22_col = fits.Column(name='s22', format='D',
-                                      array=self.s_parameters[3, 0, :])
-                col_list += [s11_col, s12_col, s21_col, s22_col]
-
-            coldefs = fits.ColDefs(col_list)
-            bandpass_hdu = fits.BinTableHDU.from_columns(coldefs)
-            bandpass_hdu.header['EXTNAME'] = 'BANDPARM'
-            hdulist.append(bandpass_hdu)
+        bandpass_col = fits.Column(name='bandpass', format='D',
+                                   array=self.bandpass_array[0, :])
+        col_list = [bandpass_col]
+
+        if self.receiver_temperature_array is not None:
+            sys_temp_col = fits.Column(name='sys_temp', format='D',
+                                       array=self.receiver_temperature_array[0, :])
```

Comment:
Is there a reason this is called a system temperature in the file, but a receiver temperature in the object? Compliance with other formats?